app/main.py

```diff
@@ -1,8 +1,51 @@
+def limitation(num: float, min_lim: float, max_lim: float) -> float:
+    if num < min_lim:
+        return min_lim
+    elif num > max_lim:
+        return max_lim
+    return num
+
+
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str) -> None:
+        self.comfort_class = limitation(comfort_class, 1, 7)
+        self.clean_mark = limitation(clean_mark, 1, 10)
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                 distance_from_city_center: float,
+                 clean_power: int,
+                 average_rating: float,
+                 count_of_ratings: int) -> None:
```

Comment:
The best approach in this case is to move all parameters on the next lines after method name